src/library_pthread.js

```diff
@@ -571,6 +572,15 @@ var LibraryPThread = {
     else postMessage({ 'cmd': 'cleanupThread', 'thread': thread });
   },
 
+  _emscripten_thread_set_strongref: function(thread) {
+#if ENVIRONMENT_MAY_BE_NODE
+    if (!ENVIRONMENT_IS_NODE) return;
+    var worker = PThread.pthreads[thread];
+    assert(worker);
```

Comment:
Nit: I'd remove this assertion and just do `PThread.pthreads[thread].ref()` inline. If there is no worker there, invoking `.ref()` will throw anyway.

Reply:
Good idea! Addressed with commit 16e738e4ad1e1a6554e7fc2662cde6435854f9eb.